src/Umbraco.Web/WebServerUtility.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Hosting;
+using Umbraco.Core;
+using Microsoft.Web.Administration;
+
+namespace Umbraco.Web
+{
+    internal class WebServerUtility
```

Comment:
I'm going to remove this class since we don't use it and then remove the Microsoft.Web.Administration; Nuget reference as well. If we need this functionality to determine bindings (which we would if we were using the server messenger for azure with web service dist calls) then we can make a package for that.